test/tests/reporter/GriddedDataReporter/dummy.i

```diff
@@ -0,0 +1,33 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+[]
+
+[Variables]
+  [u]
+  []
+[]
+
+[Kernels]
+  [null]
+    type = NullKernel
+    variable = u
+  []
+[]
+
+[Reporters]
+  [parameter_info]
+    type = GriddedDataReporter
+    parameter_file_names = 'diffusivity.txt'
+    outputs = out
+  []
+[]
+
+[Executioner]
+  type = Steady
+[]
+
+[Outputs]
+exodus=true
```

Comment:
Use csv output of the reporter or maybe it will need to be json output since it will contain a vector of vectors

Reply:
I used json and json diff for the test